### PR TITLE
feat: verify user

### PR DESCRIPTION
**Pull request by zuramai**
access `/verify` for admin to scan user QR code that is sent to their email

### CODE REVIEW

**Review by WahidinAji — APPROVED** [review]

File: frontend/components/AppHeader.vue [hunk 1]
```diff
@@ -8,6 +8,7 @@ function checkWindowSize() {
     showNavbar.value = !isMobile.value
 }
 onMounted(() => {
+    console.log("rstrst")
```
Comment (aldy505) [inline frontend/components/AppHeader.vue:11]: rstrst

File: frontend/pages/verify.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<script lang="ts" setup>
+useHead({
+    title: "Verify"
+})
+
+import {QrcodeStream} from 'vue-qrcode-reader'
+const detectedUser = ref()
+const detectedUsers = ref([])
+const onDetect = (a: any) => {
+    console.log(a)
```
Comment (aldy505) [inline frontend/pages/verify.vue:10]: This should be deleted

File: frontend/pages/verify.vue [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    detectedUser.value = {
+        name: 'Aji',
+        student: true,
+        institution: "PT Mencari Cinta Sejati"
+    }
+    // Fetching..
```
Comment (aldy505) [inline frontend/pages/verify.vue:16]: This should be implemented

**Review by aldy505 — APPROVED** [review]
After everything is addressed, you can merge it yourself if the CI is green

**Review by WahidinAji — APPROVED** [review]

**Pull request merged**